homework/index.js

```diff
@@ -28,27 +28,74 @@
     });
     return elem;
   }
+  function NewDate(dateString) {
+    const dateTime = new Date(dateString);
+    return dateTime.toLocaleString();
+  }
+
+  function renderRepoDetails(repo, section) {
+    const Mix = createAndAppend('ul', section);
+    const name = createAndAppend('li', Mix, {
+      text: 'Repository:',
+      class: 'left-side',
+    });
+    const description = createAndAppend('li', Mix, {
+      text: 'Description:',
+      class: 'left-side',
+    });
+    const fork = createAndAppend('li', Mix, {
+      text: 'Forks:',
+      class: 'left-side',
+    });
+    const time = createAndAppend('li', Mix, {
+      text: 'Updated:',
+      class: 'left-side',
+    });
 
-  function renderRepoDetails(repo, ul) {
-    createAndAppend('li', ul, { text: repo.name });
+    // now I'm going to use that name,description,fork,time in the below function
+
+    createAndAppend('a', name, {
+      href: repo.clone_url,
+      target: '_blank',
+      class: 'right-side-1',
+      text: repo.name,
+    });
+    createAndAppend('p', description, {
+      text: repo.description,
+      class: 'right-side-2',
+    });
+    createAndAppend('p', fork, { text: repo.forks, class: 'right-side-3' });
+    createAndAppend('p', time, {
+      text: NewDate(repo.updated_at),
+      class: 'right-side-4',
+    });
+  }
```

Comment:
The repository details that you render look like a table with two columns and four rows. Therefor an HTML table would be a natural choice. I noticed that you have manually fine-tuned the left padding in the classes `right-side-n` to vertically line up the values in the second column. However, this is not the recommended way as it is depended on the chosen font etc., which can lead to maintenance difficulties in the future when somebody decides the change the chosen font.

If you look closely on the UI as displayed on a mobile phone you can already see that the second column is not accurately aligned vertically:

![image](https://user-images.githubusercontent.com/2788771/65371994-4132c080-dc6a-11e9-9843-4b79708a4ba0.png)

Using an HTML table will solve this:

![image](https://user-images.githubusercontent.com/2788771/65372011-80611180-dc6a-11e9-9013-48d8f10039bb.png)

If this was an app on the App Store, which one would you install?

Reply:
now I changed my function